lago/cmd.py

```diff
@@ -154,24 +162,17 @@ def do_init(
                 template_repo_path
             )
         else:
-            try:
-                repo_name = (
-                    template_repo_name
-                    or lago.config.get('template_default_repo')
```

Comment:
About the 'template_default_repo', I'm not aware, but well, you might want to ask around (not sure if the foreman guys are still using lago or where it's being used right now). In any case, if compatibility is broken, I'd expect a major version bump.

About the 'generate-config' yes, that would be nicer, though I'd leave it for a later pr, as this one is big enough and that is not required right now, up to you though.